### PR TITLE
cn-pr-trans-referece-access-authn-authz-authorization-20180710

### PR DESCRIPTION
**Pull request by fatalc**

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
@fatalc IIRC, the translation work should be committed to https://github.com/kubernetes/kubernetes-docs-zh :)
cc @markthink

**Comment by xiangpengzhao** [comment]
hmmm, seems like that repo will not be maintained any longer...

File: content/cn/docs/reference/access-authn-authz/authorization.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+* To learn more about Authentication, see **Authentication** in [Controlling Access to the Kubernetes API](/docs/reference/access-authn-authz/controlling-access/).
+* To learn more about Admission Control, see [Using Admission Controllers](/docs/reference/access-authn-authz/admission-controllers/).
+{{% /capture %}}
+-->
+
+## 通过pod创建权限升级
+
+能够在命名空间中创建pod的用户可能会升级其在该命名空间内的权限。
+他们可以创建在该命名空间内访问其权限的pod。
+他们可以创建访问用户无法自己查看的pod，或者在具有不同/更高权限的服务帐户下运行的pod。
```
Comment (CaoShuFeng) [inline content/cn/docs/reference/access-authn-authz/authorization.md]: > They can create pods that access secrets the user cannot themselves read
> 他们可以创建访问用户无法自己查看的pod

这句把secret给翻译没了。

File: content/cn/docs/reference/access-authn-authz/authorization.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+{{< caution >}}
+**注意：** 系统管理员在授予对pod创建的访问权限时要小心。
+授予在命名空间中创建pod（或创建pod的控制器）的权限的用户可以：
+读取命名空间中的所有秘密;读取命名空间中的所有配置映射;
+并模拟命名空间中的任何服务帐户并执行帐户可以执行的任何操作。
+无论授权如何，这都适用。
```
Comment (CaoShuFeng) [inline content/cn/docs/reference/access-authn-authz/authorization.md]: 无论采用何种授权方式，这都适用。

I guess.

**Comment by zhangxiaoyu-zidif** [comment]
@xiangpengzhao ,hmm, cn translation repo is website now, that repo would not be maintained.

**Comment by zacharysarah** [comment]
@fatalc @xiangpengzhao @zhangxiaoyu-zidif 👋 The https://github.com/kubernetes/kubernetes-docs-zh repo is definitely active. Please reopen this PR there.

**Pull request closed without merging**